### ISSUE TITLE
No module named 'django'

### ISSUE THREAD

**Issue by Daemo00** [lona-web-org/lona-picocss#7]
Hi,
I am trying the application implemented in https://github.com/lona-web-org/lona/blob/d0b50b905e343c07f16fcd1e7c34af9e59e981fe/doc/content/demos/multi-user-chat/demo.py and I obtain the error in the subject when I try to run it.

I isolated the steps to reproduce it and here they are:

```shell
$ python3.11 -m venv venv
$ venv/bin/pip install lona lona-picocss
$ venv/bin/python
Python 3.11.2 (main, May 30 2023, 17:45:26) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import lona_picocss.html
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/path/to/venv/lib/python3.11/site-packages/lona_picocss/__init__.py", line 3, in <module>
    from lona_picocss.routes import IT_WORKS_ROUTE, SETTINGS_ROUTE, DEMO_ROUTES
  File "/path/to/venv/lib/python3.11/site-packages/lona_picocss/routes.py", line 1, in <module>
    from lona_picocss.views.components.typography import TypographyView
  File "/path/to/venv/lib/python3.11/site-packages/lona_picocss/views/__init__.py", line 1, in <module>
    from .django_error_views import handler403, handler404, handler500  # NOQA
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/venv/lib/python3.11/site-packages/lona_picocss/views/django_error_views.py", line 3, in <module>
    from django.shortcuts import render
ModuleNotFoundError: No module named 'django'
```

I see there is an extra `[test]` that includes Django:
https://github.com/lona-web-org/lona-picocss/blob/0ddac17b5ccf27ea4730701c54593fb012370016/pyproject.toml#L52
but if it is also mandatory to use this package, why isn't it in the core package dependencies?
Could you please have a look?

Thanks!

**Comment by fscherf** [comment]
That's a bug and is not supposed to happen. Thanks for reporting, I will fix that as soon as possible.

**Comment by fscherf** [comment]
@Daemo00: I fixed the issue and released 0.4.1. I am sorry if your first encounter with Lona ended in a bug report

**Comment by Daemo00** [comment]
Yes it is, but don't worry I still am enjoying it 😄

By the way, I have tried version `0.4.1` and can confirm this is fixed.

Thanks for the fast response and fix!
